e2e/cypress/integration/ui/settings/02_team_mgmt.spec.ts

```diff
@@ -109,19 +130,23 @@ describe('team management', () => {
     const dropdownNameUntilEllipsisLen = 25;
 
     context('when only the unassigned project is selected', () => {
-      before(() => {
+      beforeEach(() => {
         localStorage.setItem(STORE_OPTIONS_KEY, JSON.stringify([
```

Comment:
💭 I know this was here before in some way already; but generally, I think it would be preferrable if the cypress tests knew nothing about localStorage. We want projects to be selected? So we should open the picker and select them. It's probably more work, but it would improve our test robustness a lot. We wouldn't have to redo them if we decide that localStorage is not the right tool or something like that...

Reply:
That does sound better. Will take a crack in follow up PR I'm working on.